guides/release-workflow.md

```diff
@@ -0,0 +1,162 @@
+   The commands for publishing a release on PyPi are:
+
+   ```
+   $ git checkout <release_tag>
+   $ git clean -fxd
+   $ python setup.py build sdist --format=gztar
+   $ twine upload dist/*
+   ```
+
+   `Note:` The `Project Description` on PyPi is populated by the `Classifiers` in setup.py. It is useful to populate those accurately and completely. The `long dscription` field is used to populate the "Project Description" on PyPi.
+
+## Specific release procedures
+
+The following two sections provide specific instructions for releasing a python package in the two main use cases listed above. it is assumed the person doing the release uses the spacetelescope repository directly as a working directory. This means the main repository at spaetelescope nneds to be cloned and that "origin" points to spacetelescope, not to the person's fork.
+
+  ```
+  $ git clone git@github.com:spacetelescope/mypackage.git
+  $ git remote -v
+    origin git@github.com:spacetelescope/mypackage.git (fetch)
+    origin git@github.com:spacetelescope/mypackage.git (pull)
+  ```
+
+
+### Developing and releasing from the master branch.
+
+In this case all work is done on the `master` branch. Old releases are not maintained. When the code is ready for release, set the version in setup.py and set the date of the release in CHANGES.rst. Tag the release on master.
+
+  ```
+  git checkout master
+  git status
+  On branch master
+  ```
+
+Create an annotated tag which corresponds to the release and push it to the repository. Publish the release on github and PyPi.
+
+### Use a different release branch for each release.
+
+In this case the release is prepared on a dedicated branch ("release branch"). It is possible to maintain old releases by doing a patch release off a release branch. The instructions below use a third branch called "stable". It is a branch where final releases land and are tagged.
+
+The specific steps are:
+
+1. Start by editing the change log file (and possibly setup.py) to set the release version and date. Do this on the master branch.
+
+2. Prepare a release branch.
+
+   Make sure you are on the master branch. Make a branch for the release. Name the branch 2.1.3x. Always use a letter at the end of the branch name so that the version number (2.1.3) can be used later as a tag as duplicae names are harmful.
+
+   ```
+   $ git remote -v
+     origin git@github.com:spacetelescope/jwst.git (fetch)
+     origin git@github.com:spacetelescope/jwst.git (push)
+
+   $ git status # should show you are on master.
+     On branch master
+     Your branch is up-to-date with 'origin/master'.
+     nothing to commit, working tree clean
+
+   $ git branch 2.1.3x
+   $ git checkout 2.1.3x
+   ```
+
+3. Push the 2.1.3x branch to spacetelescope and create a PR to the "stable" branch. Note, that origin below refers to the spacetelescope repository. When tests pass merge with "stable".
```

Comment:
```suggestion
3. Push the 2.1.3x branch to `spacetelescope` and create a PR to the "stable" branch. Note, that origin below refers to the `spacetelescope` repository. When tests pass merge with "stable".
```

Reply:
`stable is a branch in astropy as well. Can you clarify what exactly is confusing?